packages/graphiql/src/components/ImagePreview.tsx

```diff
@@ -89,7 +89,7 @@ export class ImagePreview extends React.Component<
           ref={node => {
             this._node = node;
           }}
-          src={tokenToURL(this.props.token)}
+          src={tokenToURL(this.props.token)?.href}
```

Comment:
now that we’re introducing optional chaining, would you try building the webpack example and making sure it works and loads properly? if not we may want to tweak the example babel configs to include the plugin for this/etc if need be?

Reply:
Will try to build it and change if anything seems to be breaking. Though I don't think it should as [preset-typescript already covers it](https://github.com/babel/babel/issues/10690).

Reply:
great!